src/plugins/index_pattern_field_editor/public/components/field_format_editor/editors/number/number.test.tsx

```diff
@@ -25,7 +29,19 @@ const formatParams = {
 const onChange = jest.fn();
 const onError = jest.fn();
 
+const KibanaReactContext = createKibanaReactContext(
+  coreMock.createStart({ basePath: 'my-base-path' })
+);
+
 describe('NumberFormatEditor', () => {
+  beforeAll(() => {
+    // @see https://github.com/airbnb/enzyme/issues/1553
```

Comment:
I'm not sure what this is specifically referencing. Otherwise the code changes look good.